apps/logsdrilldown/kinds/logsdrilldown.cue

```diff
@@ -5,6 +5,8 @@ logsdrilldownv0alpha1: {
 	schema: {
 		spec: {
 			defaultFields: [...string] | *[]
+			prettifyJSON: bool
+			wrapLogMessage: bool
 		}
```

Comment:
defaultFields would be a general-purpose settings and prettifyJSON and wrapLogMessage user preferences. How do you think these would be used by the plugin?